airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -316,7 +316,7 @@ def _field_to_datetime(value: Union[int, str]) -> pendulum.datetime:
     def _filter_old_records(self, records: Iterable) -> Iterable:
         """Skip records that was updated before our start_date"""
         for record in records:
-            updated_at = record[self.updated_at_field]
+            updated_at = record.get(self.updated_at_field)
```

Comment:
Why do we specify such a safe data retrieval method, we may receive a record that does not contain the `updated_at_field` field

Reply:
Reverted it. I changed because i got an issue calling data from our hubspot instance. Therefore i changed it to .get. But i reran it with a smaller timeframe with the reverted change. No issues. So it seems The issue was in our hubspot instance.